capstone/src/main/webapp/index.html

```diff
@@ -1,96 +1,142 @@
+                arcu, dictum at justo ut, gravida maximus nibh. Proin id mollis
+                magna. Donec ultricies finibus ante, at vestibulum elit volutpat
+                vel. Fusce felis arcu, malesuada finibus convallis a, vulputate
+                ac tellus. Quisque diam nulla, scelerisque eu mauris ut,
+                vehicula ornare diam.
+              </p>
+            </div>
           </div>
-           <div class="col s12 tb home" id="get_started">
-            <h3 class=" white-text text-darken-4">Get Started</h3>
-            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
-                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
-                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
-                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.</p>
+          <div class="col s12 tb home" id="get_started">
+            <div class="tab-container">
+              <h3 class="white-text text-darken-4">Get Started</h3>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut
+                pharetra feugiat vehicula. Morbi vitae varius felis. Ut quis
+                interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex
+                arcu, dictum at justo ut, gravida maximus nibh. Proin id mollis
+                magna. Donec ultricies finibus ante, at vestibulum elit volutpat
+                vel. Fusce felis arcu, malesuada finibus convallis a, vulputate
+                ac tellus. Quisque diam nulla, scelerisque eu mauris ut,
+                vehicula ornare diam.
+              </p>
+            </div>
           </div>
-        </div>   
-      </div>   
-    </section>  
-
-    <footer class="page-footer">
-       <div>
-           © 2020 Studentia inc
-       </div>
-    </footer>
+        </div>
+      </section>
+      <footer class="page-footer">
+        <div>
+          © 2020 Studentia inc
+        </div>
+      </footer>
+    </div>
 
     <!--JavaScript at end of body for optimized loading-->
-    <script type = "text/javascript" src = "https://code.jquery.com/jquery-2.1.1.min.js"></script>         
-    <script type="text/javascript" src = "https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"></script> 
-     <script src="script.js"></script>    
+    <script
+      type="text/javascript"
+      src="https://code.jquery.com/jquery-2.1.1.min.js"
+    ></script>
+    <script
+      type="text/javascript"
+      src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"
+    ></script>
+    <script src="script.js"></script>
+    <div class="sidenav-overlay"></div>
```

Comment:
Nit: move these divs above the script tags

Reply:
removed them they didn't add to functionality